.github/workflows/release.yaml

```diff
@@ -21,14 +21,21 @@ jobs:
             curl \
             jq \
             squashfs-tools \
-            xz-utils
+            xz-utils \
+            gawk
```

Comment:
I think we also need to install yq here, is there a package? Otherwise one could also use `grep` ;)

Reply:
yq is already available. 
Tested and it is working.